caput/pipeline.py

```diff
@@ -1263,7 +1266,13 @@ def read_process_write(self, input, input_filename, output_filename):
             output_dirname = os.path.dirname(output_filename)
             if not os.path.isdir(output_dirname):
                 os.makedirs(output_dirname)
-            self.write_output(output_filename, output)
+            _OneAndOne.write_output(
```

Comment:
Revert this back to `self.write_output`, if `self.` will work with a `staticmethod`.

or `self.__class__.write_output`

Reply:
It does!